src/storage/heap_file.c

```diff
@@ -22486,6 +22737,12 @@ heap_insert_logical (THREAD_ENTRY * thread_p, HEAP_OPERATION_CONTEXT * context,
       goto error;
     }
 
+/*SUPPLEMENT_INSERT REDO LSA, INSERT REC_BIGONE  */
+  if (context->do_supplemental_log && heap_is_big_length (context->recdes_p->length))
+    {
+      LSA_COPY (&context->supp_redo_lsa, &tdes->tail_lsa);
```

Comment:
I recommend you using `logtb_find_current_tran_lsa()` instead of directly accessing `tdes->tail_lsa`. It also make you to be able to get rid of `tdes = LOG_FIND_CURRENT_TDES (thread_p);`s. This could be applied to all heap operations.

Reply:
refer to `heap_log_delete_physical()`